calico_node/Makefile

```diff
@@ -6,45 +6,63 @@ CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)
 CALICO_NODE_DIR=$(dir $(realpath $(lastword $(MAKEFILE_LIST))))
 VERSIONS_FILE?=$(CALICO_NODE_DIR)/../_data/versions.yml
 
-# Read current stream version from _data/versions.yml
-RELEASE_STREAM?=$(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - "currentReleaseStream")
-
 ###############################################################################
 # Determine whether there's a local yaml installed or use dockerized version.
 # Note in order to install local (faster) yaml: "go get github.com/mikefarah/yaml"
 YAML_CMD:=$(shell which yaml || echo docker run --rm -i $(CALICO_BUILD) yaml)
 
-CALICO_VER ?= $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].title')
```

Comment:
I noticed that there was a long fixed startup time each time I ran `ST_TO_RUN=... make st`, and specifically about 30 times running the go-build container, each for extracting a single value from versions.yml. This is because those invocations are on the RHS of = variable definitions - which means they run every time each variable is used - whereas with := it's just once when the variable is defined.